changelogs/current.yaml

```diff
@@ -186,6 +186,9 @@ new_features:
 - area: dubbo_proxy
   change: |
     added :ref:`metadata_match <envoy_v3_api_field_extensions.filters.network.dubbo_proxy.v3.RouteAction.metadata_match>` support to the dubbo proxy.
+- area: cors
+  change: |
+    added support for cors PNA. This behavioral change can be temporarily reverted by setting runtime guard ``envoy_reloadable_features_cors_private_network_access`` to false.
```

Comment:
Please add a link to `https://developer.chrome.com/blog/private-network-access-preflight/`

Reply:
Added.